packages/components/Button/SPEC.md

```diff
@@ -70,14 +73,26 @@ The `Button` component can include an `icon` that appears before or after its `c
 
 The `Button` component supports different sizing with at least three different sizes: `small`, `medium`, and `large`. `Small` is the default on `win32`, `medium` is the default on other platforms.
 
+#### Size modification is not explicitly supported on Android and is provided as an 'as-is' functionality.
```

Comment:
I think instead of this being a header it would make more sense to bold this text. In markdown I think that's wrapping it in two asterisks (**)

See https://www.markdownguide.org/cheat-sheet

You could also then remove the `</br>`s

Reply:
Same comment for the other headers you've added

Reply:
Alright